client/src/components/Messages/Message.jsx

```diff
@@ -80,6 +94,49 @@ export default function Message({
 
   const wrapText = (text) => <TextWrapper text={text} />;
 
+  const resubmitMessage = () => {
+    const text = textEditor.current.innerHTML
```

Comment:
innerHTML adds the HTML tags to the visible text and API response: 
`<div>How do I make modular code in python? what is the convention</div>`

Changing this to `innerText` fixes it

Reply:
fixed